src/controllers/index.js

```diff
@@ -3,22 +3,31 @@ import home from './home';
 import { getLogin, postLogin } from './login';
 import logout from './logout';
 import events from './events';
+import bookings from './bookings';
+import { phoneVerifyStart } from './phone_verification';
 import eventDetails from './eventDetails';
 import checkId from './checkNumber';
 import validLogin from './valid_login';
 import adminEvents from './admin_events';
 import adminCourses from './admin_courses';
 
-const router = express.Router();
+const controler = express.Router();
 
-router
+controler
   .get('/', home)
-  .get('/admin/login', getLogin)
-  .post('/admin/login', validLogin, postLogin)
-  .get('/admin/logout', logout)
   .get('/events', events)
-  .get('/admin/events', adminEvents)
-  .get('/admin/courses', adminCourses)
+  .get('/admin/login', getLogin)
   .get('/courses', events)
-  .get('/eventdetails/:id', checkId, eventDetails);
-export default router;
+  .get('/bookings', bookings)
+  .post('/phone/verification/start', phoneVerifyStart)
+  .get('/eventdetails/:id', checkId, eventDetails)
+  
+const adminControler = new express.Router();
+
+adminControler
+  .post('/login', validLogin, postLogin)
```

Comment:
api/v1/login

Reply:
I think this is a mistake of mine to make a new instance of the Router
I thought that I need that in order to use a middleware authentication
- I will remove the second adminControler and I'll remove it from app.js
- instead later we will use router.use('admin', someMiddleware ) to filter all /admin routes